jbpm/jbpm-flow/src/main/java/org/jbpm/process/core/timer/BusinessCalendarImpl.java

```
@@ -338,6 +336,12 @@ protected int getPropertyAsInt(String propertyName, String defaultValue) {
         return Integer.parseInt(value);
     }
 
+    protected int getPropertyAsInt(String propertyName) {
+        String value = businessCalendarConfiguration.getProperty(propertyName);
+
+        return Integer.parseInt(value);
```

We might want to return a nicer error message if this fails:
```
Caused by: java.lang.NumberFormatException: For input string: ""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:678)
	at java.base/java.lang.Integer.parseInt(Integer.java:786)
	at org.jbpm.process.core.timer.BusinessCalendarImpl.getPropertyAsInt(BusinessCalendarImpl.java:342)
```